spec/index.html

```diff
@@ -495,6 +498,31 @@ <h2>RDF Documents and Syntaxes</h2>
     and still the subject of group discussion. An alternative to conformance
     levels, "profiles", may be adopted instead, abandoned, or described in 
     another specification.</p>
+
+  <section id="rdf-strings">
+    <h2>Strings in RDF</h2>
+
+    <p>RDF uses Unicode [[Unicode]] as the fundamental representation for string values.
+      Within this, and related specifications, the term <dfn id="dfn-rdf-string">string</dfn>,
+      or <a data-lt="string">RDF string</a>,
+      is used to describe an ordered sequence of zero or more
+      <a data-lt="code point" class="lint-ignore">Unicode code points</a>
+      which are <a data-cite="i18n-glossary#dfn-scalar-value" class="lint-ignore">Unicode scalar values</a>.
+      Unicode scalar values do not include the
+      <a data-cite="i18n-glossary#dfn-surrogate" class="lint-ignore">surrogate code points</a>.
+      Note that most <a>concrete RDF syntaxes</a> require the use
+      of the UTF-8 character encoding [[!RFC3629]], 
+      which requires that certain non-character values be expressed
+      using the `\u0000` or `\U00000000` forms.
```

Comment:
This might clarify that the UCHAR forms are part of the concrete RDF syntaxes, and not UTF-8.

```suggestion
      Note that most <a>concrete RDF syntaxes</a> require the use
      of the UTF-8 character encoding [[!RFC3629]], 
      and use the `\u0000` or `\U00000000` forms to express certain non-character values.

Reply:
One nit: some of the "non-character values" are not non-characters. For example, NULL (U+0000) is a real Unicode character. I might not change the text here, though, because the characters that require escapes are non-XML characters and many of these are _also_ non-character code points. (Also, the escapes can be used for "normal" characters and can be especially helpful for e.g. invisible characters such as the bidi controls)

@pfps I read the above proposed change as two useful things: most syntaxes require UTF-8 (which is a fact), thus eliminating non-Unicode character encoding forms. And most syntaxes use escapes to express the above mentioned code points.